R/analyse.R

```diff
@@ -35,6 +35,16 @@ analyse <- function(config) {
   run_pacta(config)
   run_aggregate_alignment_metric(config)
   plot_aggregate_loanbooks(config)
+  
+  write_manifest(
+    config = config,
+    path = file.path(output_analysis_dir, "manifest.yml"),
+    prior_input_paths = c(
+      get_output_prepare_dir(config),
+      get_output_matched_loanbooks_dir(config),
```

Comment:
`analyse()` does not technically load `matched_loanbooks` for any of the functions it calls. I am a bit on the fence on whether that should mean that we do not add the timestamp of `matched_loanbooks` (no direct dependency and you can backtrace the timestamp by checking `prioritized_loanbooks_and_diagnostics`, which does depend on `matched_loanbooks`) or we do add it (because it is impossible to run `analyse()` without having previously run `match_loanbooks()`...

Reply:
also on the fence, so I went for more information instead of less?

Reply:
let's keep it for now, probably no reason not to